validator/gulpjs/test/validate.js

```diff
@@ -0,0 +1,196 @@
+/**
+ * @license
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the license.
+ */
+
+'use strict';
+
+const assert = require('assert');
+const es = require('event-stream');
+const gutil = require('gulp-util');
+const File = require('vinyl');
+const fs = require('fs');
+const amphtmlValidator = require('../');
```

Comment:
'amphtmlValidator' could be a tad confusing here because it's suggestive of the 'amphtml-validator' module. I know it's even longer, almost JavaEsque but maybe calling these things 'gulpAmphtmlValidator' would be better, since that's more like the name of the module? If not, maybe just 'validator' is good also?
Please feel free to apply this comment consistently to the other files as well.